lib/Http/Client/RequestClient.js

```diff
@@ -18,6 +18,7 @@ const convertHttpRequest = function convertHttpRequest(req) {
         method: req.method,
         headers: req.headers,
         followAllRedirects: true,
+        encoding: req.encoding
```

Comment:
Confirmed that this does not affect JSON body responses when `req.encoding` is undefined